src/common/lib/led_helper.h

```
@@ -85,6 +85,47 @@ extern volatile bool cmd_mode_led_green;
+ * // Apple M0110 protocol (uses >= comparison):
+ * update_keyboard_ready_led(keyboard_state >= INITIALISED);
+ * ```
+ * 
+ * @param ready true if keyboard is initialized and ready, false otherwise
+ * 
+ * @note Inline function provides zero-overhead abstraction
+ * @note Compiled to nothing when CONVERTER_LEDS is not defined
+ * @note ISR-safe: update_converter_status() only sets flags, no blocking I/O
+ */
+static inline void update_keyboard_ready_led(bool ready) {
+#ifdef CONVERTER_LEDS
```

Clang (14.0.6)

[warning] nested redundant `#ifdef`; consider removing it

(readability-redundant-preprocessor)